src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -194,7 +198,7 @@ public void onFailure(Throwable t) {
                             LOGGER.log(WARNING, "Failed to load " + e, t);
                         }
                     }
-                });
+                }, newExecutorService());
```

Comment:
Not sure I get it. The original code was not specifying an executor service, so why do we need to _add_ this code now?

Reply:
Deprecated in v22 https://github.com/google/guava/blob/v22.0/guava/src/com/google/common/util/concurrent/Futures.java#L1073

Removed in v26
https://github.com/google/guava/blob/v26.0/guava/src/com/google/common/util/concurrent/Futures.java (not there anymore)

Reply:
https://github.com/google/guava/blob/v22.0/guava/src/com/google/common/util/concurrent/Futures.java#L1158-L1167 I guess you mean. :+1: